LICENSE.md

```diff
@@ -4,7 +4,7 @@ Copyright (c) 2010 Tim Caswell
 
 Copyright (c) 2014 Jordan Harband
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of
+Permission is + granted, free of charge, to any person obtaining a copy of
```

Comment:
I'm very confused about the point of this change. "hereby" is correct, the + is not.

Reply:
Yes, "hereby" will not be understood as much by the community.

Reply:
@Yash-Singh1 anyone who doesn't know the term "hereby" is not going to be qualified (in a legal sense) to understand any project license written in English, so I'm not sure to which community you're referring.